content/docs/setup/behavioral_changes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Behavioral Changes in 1.1
+description: Major changes in behavior, both backward compatibility breaking and non, in Istio 1.1.
+weight: 15
+icon: setup
+
+---
+
+# Major Behavioral Changes in Istio 1.1
+
+## Installer Changes
+
+### Changes with upgrade impact
+
+- Istio’s CRDs have been placed into their own Helm chart `istio-init`.  This prevents loss of custom resource data, facilitates the upgrade process, and enables the Istio project to evolve beyond a Helm-based installation.  The [upgrade documentation](/docs/setup/kubernetes/upgrade/) provides the proper procedure for upgrading from Istio 1.0.6 to Istio 1.1.0.  Please follow these instructions **carefully and precisely** when upgrading.  If certmanager is desired, it is mandatory to use the `--set certmanager=true` flag when installing both istio-init and istio charts with either `template` or `tiller` installation modes.
+- The 1.0 istio-remote chart used for [multicluster VPN](/docs/setup/kubernetes/install/multicluster/vpn/) and [multicluster split horizon](/docs/examples/multicluster/split-horizon-eds/) remote cluster installation has been consolidated into the Istio chart.  To generate an equivalent istio-remote chart, use the flag `--set global.istioRemote=true`.
+- Addons are no longer be exposed via separate load balancers.  Instead addons are exposed via the ingressgateway.  To expose an addon via the ingressgateway, please follow the [Remotely Accessing Telemetry Addons](/docs/tasks/telemetry/gateways/) guide.
+- The built-in Istio statsd collector has been removed.  Istio retains the capability of integrating with your own statsd collector.
+- Grafana, Prometheus, Kiali, and Jaeger passwords and username are now stored in [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/) instead of command line configuration options, values.yaml, or configmaps for improved security and compliance.
+- Jaeger has replaced Zipkin as the default tracing system.
+- The “ingress” series of options for configuring a Kubernetes Ingress have been removed.  Kubernetes Ingress is still functional and can be installed by following the [Securing Kubernetes Ingress with Cert-Manager](docs/examples/advanced-gateways/ingress-certmgr/) guide.
+
+### Changes without upgrade impact
+
+- Several installation profiles have been added to simplify the installation process.  To use this feature, read the [Helm Installation Instructions](/docs/setup/kubernetes/install/helm/) and select a profile from the document tabs.  This feature enables a better user experience by simplifying the installation process for several well-known patterns.
```

Comment:
Profiles link is `https://preliminary.istio.io/docs/setup/kubernetes/additional-setup/config-profiles/`.  This section needs to be re-worded th4e match the proper link.

Reply:
no longer relevant.  I removed this section since it's redundant with the release notes